watertap/examples/flowsheets/RO_with_energy_recovery/RO_with_energy_recovery.py

```diff
@@ -104,38 +128,44 @@ def build():
     m.fs.product = Product(default={"property_package": m.fs.properties})
     m.fs.disposal = Product(default={"property_package": m.fs.properties})
 
-    # costing
+    # costing for all configurations
     m.fs.P1.costing = UnitModelCostingBlock(
         default={"flowsheet_costing_block": m.fs.costing}
     )
```

Comment:
If we're going to create the costing blocks immediately after the unit, let's just do that for `P1` too.